examples/kubernetes/1.10/cilium-crio.yaml

```diff
@@ -476,6 +476,7 @@ metadata:
   labels:
     io.cilium/app: operator
     name: cilium-operator
+    k8s-app: cilium-operator
```

Comment:
I think this will break the "well-known" identity for cilium operator? cc: @aanm , @tgraf .

Reply:
Correct

Reply:
OK - reverting approval in that case. @eloycoto please remove the change in the labels from this PR, and instead wait for the operator pods to be ready using `-l name=cilium-operator` in the following commit.

Reply:
@eloycoto please remove `k8s-app: cilium-operator` from the set of labels here and from the other YAMLs in this commit.